cms/djangoapps/api/v1/tests/test_views/test_course_runs.py

```diff
@@ -138,44 +138,76 @@ def test_update_with_invalid_user(self):
         assert response.status_code == 400
         assert response.data == {'team': [{'user': ['Object with username=test-user does not exist.']}]}
 
-    def test_partial_update(self):
+    def test_role_get_or_create(self):
+        """
+        Verify that the api gets or creates course access role
+        """
```

Comment:
`Tests that when the course run is updated with instructor data, it does not create multiple roles. `